src/sentry/snuba/subscriptions.py

```diff
@@ -111,10 +111,21 @@ def update_snuba_subscription(subscription, query, aggregation, time_window, res
     return subscription
 
 
+def bulk_delete_snuba_subscriptions(subscriptions):
+    """
+    Deletes a list of snuba query subscriptions.
+    :param subscriptions: The subscriptions to delete
+    :return:
+    """
+    for subscription in subscriptions:
+        # TODO: Batch this up properly once we move to tasks.
+        delete_snuba_subscription(subscription)
```

Comment:
In the interim do you need any retries or error handling here?

Reply:
I'm not concerned with things ending up in a bad state at the moment since Snuba doesn't actually even do anything.